syntax = "proto3";

option java_multiple_files = true;
option java_package = "cn.tansorflow.server.protocol";

message ClientInfo{
  string serverInfo=1;
  string key=2;
}
message MyResponse{
  int32 code=1;
  string msg=2;
}

service UpdateServerInfo {
  rpc updateInfo(ClientInfo) returns (MyResponse) {}
}
